providers/manage.rb

```diff
@@ -118,11 +118,23 @@ def chef_solo_search_installed?
         end
 
         if u['ssh_keys']
+          #on suse, i had errors if group didn't exist with the ssh_keys template, so try creating first
+          case node['platform_family']
+          when 'suse'
+            group new_resource.group_name do
+              gid new_resource.group_id
+              action :create
+            end
+          end
```

Comment:
yeah - definitely looks cleaner! 
Thanks for the Ruby syntax tips, as I'm coming from a Java world... : )

How can I help test the other platforms? I read [somewhere] (https://github.com/opscode-cookbooks/users/blob/master/CONTRIBUTING.md#functional-and-unit-tests) that this cookbook uses Test Kitchen, but then I couldn't find the .kitchen.yml file... ?